docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -296,6 +296,21 @@ more information.)
 ![DynamoDB local connection in NoSQL Workbench](../../img/database-access/nosql-workbench-connection.png)
 
 
+## SQL Server Azure Data Studio
+
+In Azure Data Studio create a connection using your proxy's endpoint. This is
+`localhost,62652` in the example above. Using the IP `127.0.0.1,62652` host could be required on
+a Windows machine instead of `localhost`. (See
+[Get connection information](./gui-clients.mdx#get-connection-information) for
+more information.)
```

Comment:
```suggestion
`localhost,62652` in the example above. On a Windows machine, using an address in the format `127.0.0.1,62652` could be required instead of `localhost`. (See
[Get connection information](./gui-clients.mdx#get-connection-information) for
more information.)
```

Reply:
Does the `tsh proxy db` command indicate whether you can use `localhost` or not? If that's the case, maybe we can say so explicitly instead of linking to the "Get connection information" section.